gemfiles/1.9.2-Gemfile

```diff
@@ -0,0 +1,12 @@
+source "https://rubygems.org"
+
+gem 'faraday', '>= 0.8.7'
+gem 'faraday_middleware', '>= 0.9.2'
+gem 'rack', "1.4.5" # Need to lock rack to an old version for old rubies
+
+# dev dependencies
+gem 'json', "~> 1.8.0"
+gem "bundler", "~> 1.3"
+gem "rspec", "~> 3.0"
+gem "webmock", "1.24.6" # last version supported by 1.9.2
+gem "rake", "0.9.6" # last version supported by 1.9.2
```

Comment:
Yeah, my comment isnt exactly accurate. It should have been "a version known to support 1.9.2" - not necessarily the last. I guessed at one and tried it.